eng/versioning/version_client.txt

```diff
@@ -20,13 +20,13 @@ com.azure:azure-security-keyvault-secrets;4.0.1;4.1.0-beta.1
 com.azure:azure-messaging-eventhubs;5.0.0-beta.6;5.0.0-beta.7
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-sdk-template;1.0.4-beta.2;1.0.4-beta.2
-com.azure:azure-storage-blob;12.1.0;12.2.0-beta.1
+com.azure:azure-storage-blob;12.2.0-beta.1;12.2.0-beta.2
 com.azure:azure-storage-blob-batch;12.1.0;12.2.0-beta.1
 com.azure:azure-storage-blob-cryptography;12.1.0;12.2.0-beta.1
-com.azure:azure-storage-common;12.1.0;12.2.0-beta.1
-com.azure:azure-storage-file-share;12.0.0;12.1.0-beta.1
-com.azure:azure-storage-file-datalake;12.0.0-beta.7;12.0.0-beta.8
-com.azure:azure-storage-queue;12.1.0;12.2.0-beta.1
+com.azure:azure-storage-common;12.2.0-beta.1;12.2.0-beta.2
+com.azure:azure-storage-file-share;12.1.0-beta.1;12.1.0-beta.2
+com.azure:azure-storage-file-datalake;12.0.0-beta.8;12.0.0-beta.9
```

Comment:
Weren't you guys considering ticking up the minor version of datalake just so when you GA it'll sort higher than the previous preview versions? I'm going to approve the PR as is since everything looks fine and the suggested version increase for datalake was just that, a suggestion.

Reply:
We're not sure what the next release is, so I'll leave it as 12.0.0 for now and when we're ready to GA I'll adjust the version appropriately.